packages/block-library/src/navigation/edit.js

```diff
@@ -71,18 +71,18 @@ function Navigation( {
 				return null;
 			}
 
-			return pages.map( ( { title, type, link: url, id } ) => (
-				createBlock( 'core/navigation-link',
+			return pages.map( ( { title, type, link: url, id } ) => {
+				return createBlock( 'core/navigation-link',
 					{
 						type,
 						id,
 						url,
-						label: escape( title.rendered ),
-						title: escape( title.raw ),
+						label: title.rendered === '' ? __( '(no title)' ) : escape( title.rendered ),
+						title: title.raw === '' ? __( '(no title)' ) : escape( title.raw ),
```

Comment:
Are the `title.rendered` and `title.raw` properties always defined? If not, it would be probably safer not to compare it explicitly to an empty string.

Reply:
these should be definded as they always come with the post, however, you are right that comparing to empty string is not the best approach. :)